CPT331.WebAPI/CPT331.WebAPI.csproj

```diff
@@ -125,8 +143,27 @@
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
+    <Content Include="Content\bootstrap-theme.css" />
```

Comment:
Are all these files added by Swagger? There seems to be a lot of JavaScript and other such stuff, none of which we really need in a web API project.